UI/Web/src/app/cards/card-item/card-item.component.ts

```diff
@@ -159,6 +160,7 @@ export class CardItemComponent implements OnInit, OnDestroy {
 
     if (this.utilityService.isChapter(this.entity)) {
       this.chapterTitle = this.utilityService.asChapter(this.entity).titleName;
+      this.volumeTitle = this.utilityService.asChapter(this.entity).volumeTitle || '';
```

Comment:
Remove this. Refactor chapterTitle -> tooltipTitle and have logic to show as either VolumeTitle or ChapterTitle. This will lead to a streamlined implementation.

Reply:
Please note, that chapterTitle can be the Title from ComicInfo/Epub and that should take priority over Chapter X or Volume Y Chapter X.

Reply:
Otherwise, since we are moving into release testing, we can hold the volume Y Chapter X changes and split into a separate PR and work that, so your other work can be merged in this release.

Reply:
I have moved the Volume information to the tootlip as suggested. I think I still need this line to populate the property though.

Reply:
What I was wanting is the logic to generate the title would be consolidated into one variable and done here. Then the getter (get tooltip) would return the title itself. This would simplify the component.

Reply:
The new update has consolidated tooltipTitle into one variable.